doc/common/README.md

```diff
@@ -569,14 +569,51 @@ d. Участники, проверяющие код, добавляют ком
 
 e. Комментарии отмечает выполненными автор изменений, код которого просматирвают.
 
-f. После корректировки замечаний и реализации предложений должен запрашиваться
+f. В начале процесса исправления замечаний задача переводится в соотвествующий рабочий статус. 
+   После завершения исправлений задача переводится обратно в статус ревью.
+
+g. После корректировки замечаний и реализации предложений должен запрашиваться
 повторный обзор кода.
 
-g. Все вопросы по ревью обсуждаются наибыстрейшим из возможных способов. Недопустимо
+h. Все вопросы по ревью обсуждаются наибыстрейшим из возможных способов. Недопустимо
 использовать такой способ коммуникации, при котором ответ затягивается, а соответственно
 затягивается и ревью.
 
-#### 6.4.4. Противоречия и споры в процессе обзора кода
+#### 6.4.4. Процесс обзора задачи, не связанных с системой управления кодом (Github, Gitlab)
+
+Такие задачи, как документирование в Notion, Confluence и пр. 
+
+Мотивация:\
+Сократить количество отвлекающих воздействий на автора изменений до 2 (в позитивном сценарии), 
```

Comment:
We can solve it using mode `Not Disturb Me` (https://get.slack.help/hc/en-us/articles/214908388-Pause-notifications-with-Do-Not-Disturb) in Slack with specified schedule. We can create special rule for our team so that we have only a few time windows when we check out new messages and send responses.